homeassistant/components/verisure/__init__.py

```diff
@@ -34,7 +34,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     coordinator = VerisureDataUpdateCoordinator(hass, entry=entry)
 
     if not await coordinator.async_login():
-        raise ConfigEntryAuthFailed
+        raise ConfigEntryNotReady
```

Comment:
Adding a small message to it will help the user (it shows up in the integration dashboard).